oauth/oauth.go

```diff
@@ -27,15 +27,20 @@ type Config struct {
 	RefreshToken string
 }
 
-// NewConfig creates a new *Config for the provided client credentials.
+// NewConfig creates a new *Config for the provided client credentials and default token URL.
 func NewConfig(clientID, clientSecret, refreshToken string) *Config {
+	return NewConfigWithBaseUrl(clientID, clientSecret, refreshToken, TokenURL)
+}
+
+// NewConfig creates a new *Config for the provided client credentials and token URL.
+func NewConfigWithBaseUrl(clientID, clientSecret, refreshToken, tokenUrl string) *Config {
 	return &Config{
 		Config: &oauth2.Config{
 			ClientID:     clientID,
 			ClientSecret: clientSecret,
 			Endpoint: oauth2.Endpoint{
 				AuthStyle: oauth2.AuthStyleInParams,
-				TokenURL:  TokenURL,
+				TokenURL:  tokenUrl,
```

Comment:
Same as in `client.go` for consistency: `NewConfigWithBaseUrl` => `NewConfigWithBaseURL` and `tokenUrl` => `tokenURL`.